src/base/modes.scss

```diff
@@ -3,9 +3,12 @@
 @import "../support/mixins/color-modes.scss";
 
 @import "@primer/primitives/dist/scss/colors/_light.scss";
+@import "@primer/primitives/dist/scss/colors/_light_protanopia.scss";
```

Comment:
It seems this file is not available. @jonrohan Shouldn't [`@primer/primitives@4.8.0`](https://github.com/primer/primitives/pull/237) get picked up automatically after changing to the `^` in https://github.com/primer/css/pull/1594?

Reply:
The change still only uses the one specified in package.json, but it looses the restriction of a codebase that requires both `primer/css` and `primer/primitives`. 

Basically telling npm, _I use 4.7.1, but if someone is using me and needs 4.8.0 then I can use that also._

I've pushed a commit to update to 4.8.0 in this pr to fix the problem.